data/src/main/java/org/apache/iceberg/data/avro/GenericReaders.java

```diff
@@ -111,19 +111,19 @@ public OffsetDateTime read(Decoder decoder, Object reuse) throws IOException {
   }
 
   private static class GenericRecordReader extends ValueReaders.StructReader<Record> {
-    private final StructType struct;
+    private final StructType record;
```

Comment:
What about structType instead? In the context of a record reader, record sounds like it would not be a type.

Reply:
Makes sense!